ToDoApp/ToDoApp/CreateTaskView-ViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CreateTaskView-ViewModel.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+
+extension CreateTaskView{
+    @MainActor class CreateTaskViewModel: ObservableObject{
+        @Published var title: String = "Default Title"
+        @Published var titleInput: String = ""
+        @Published var dateInput: Date?
```

Comment:
let's keep only this two properties. there is no need in title, selectedDates, mydates

Reply:
let's make them readonly as well since we don't need them to be modified from the outside world because we don't want to have side effects available and we will follow the encapsulation principle
 ```suggestion
        @Published private(set) var titleInput: String = ""
        @Published private(set) var dateInput: Date?
```

Reply:
This has been fixed.